### PR TITLE
Enhancements to ibm block storage csi driver

### PR DESCRIPTION
**Pull request by ArbelNathan**
* changed offering name 
* added logo
* added missing metadata fields

### CODE REVIEW

**Comment by nkkashyap** [comment]
Few more comments
1. In `custom-parameters.json` 
    - update `"category": "text"` to `"category": "config"`

2. In `storage-class-parameters.json`
    - for `allowVolumeExpansion?` update  `"category": "boolean"` to `"category": "config"`  and add `"type": "boolean"

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.4.0/metadata.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      "name": "volume-encryption-in-transit",
+      "value": "false"
+    },
+    {
+      "name": "volume-snapshot",
+      "value": "true"
+    },
+    {
+      "name": "volume-availability",
+      "value": "regional"
```
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply config-templates/ibm/ibm-system-storage-block-csi-driver/1.4.0/metadata.json:60]: correct - you can create a mounting pod for the volume on any worker node in the cluster

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.4.0/metadata.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      "name": "supported-envs",
+      "value": "IBM"
+    },
+    {
+      "name": "supported-k8s-versions",
+      "value": "1.18,1.19"
+    },
+    {
+      "name": "volume-type",
+      "value": "block,file"
```
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply config-templates/ibm/ibm-system-storage-block-csi-driver/1.4.0/metadata.json:44]: It supports block storage and can mount `block` and `file systems` on volumes.. so which one do you mean?

[the remark this answers is not in the document]
Comment (nkkashyap) [reply config-templates/ibm/ibm-system-storage-block-csi-driver/1.4.0/metadata.json:44]: Okay it is block and supports both Block and FileSystem `volumeMode`.
 The `volume-type` should be `block` only.

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.4.0/metadata.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply config-templates/ibm/ibm-system-storage-block-csi-driver/1.4.0/metadata.json:60]: but only one at a time

File: config-templates/template_list.json [hunk 3]
```diff
@@ -70,5 +70,13 @@
     "default-version": "4.5",
     "enabled": "true",
     "provider": "redhat"
+  },
+  {
+    "name": "ibm-system-storage-block-csi-driver",
+    "displayname": "IBM block storage CSI driver",
```
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply config-templates/template_list.json]: done

File: config-templates/template_list.json [hunk 4]
```diff
@@ -70,5 +70,13 @@
...
+    "Description": "IBM block storage CSI driver for IBM Spectrum Virtualize, IBM FlashSystem and IBM DS8000 Family.",
```
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply config-templates/template_list.json]: done

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.4.0/metadata.json [hunk 5]
```diff
@@ -0,0 +1,75 @@
+{
+  "name": "1.4.0",
+  "enabled": "true",
+  "description": "IBM block storage CSI driver version 1.4.0",
```
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply config-templates/ibm/ibm-system-storage-block-csi-driver/1.4.0/metadata.json]: done

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.4.0/metadata.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (ArbelNathan) [reply config-templates/ibm/ibm-system-storage-block-csi-driver/1.4.0/metadata.json]: done

**Review by nkkashyap — APPROVED** [review]
Looks good

**Pull request merged**